pandas/core/internals/managers.py

```diff
@@ -1013,6 +1013,9 @@ def value_getitem(placement):
                 return value
 
         else:
+            if value.ndim == 2:
+                value = value.T
+
             if value.ndim == self.ndim - 1:
```

Comment:
this can become an elif

Reply:
indeed, will also include in a next PR

Reply:
Actually, it needs to be an `if`, because this case needs to end up in the final `else` to define the `value_getitem` function